rust/private/rustc.bzl

```diff
@@ -671,7 +671,14 @@ def collect_inputs(
     """
     linker_script = getattr(file, "linker_script") if hasattr(file, "linker_script") else None
 
-    linker_depset = cc_toolchain.linker_files()
+    # TODO: As of writing this comment Bazel used Java CcToolchainInfo.
```

Comment:
Do you mind filing a Github issue and put it here so we can triage later? It's hard to keep track of all the TODOs.

Reply:
Done.